math/fibonacci.c

```diff
@@ -1,23 +1,40 @@
+/**
+ * @file fibonacci.c
+ * @brief Program to print the nth term of the Fibonacci series.
+ * @details Fibonacci series generally starts from 0 and 1. Every next term in
+ *the series is equal to the sum of the two preceding terms.
+ **/
+
 #include <stdio.h>
 
-// Fibonnacci function
-int fib(int number)
-{
-    if (number == 1 || number == 2)
-        return 1;
-    else
-        return fib(number - 1) + fib(number - 2);
-}
+unsigned fib(int);
 
+/* Driver Code */
 int main()
```

Comment:
This code was written a long time ago, since it looks like you are trying to improve it, you might want to add a `test()` to bring it up to the current standard

Reply:
...I see that you probably didn't read the contribution guidelines carefully, [here's the link](https://github.com/TheAlgorithms/C/blob/master/CONTRIBUTING.md)